distribution/pom.xml

```diff
@@ -119,6 +141,9 @@
                         <link>http://lucene.apache.org/core/5_5_4/analyzers-common/</link>
                         <link>http://lucene.apache.org/core/5_5_4/queryparser/</link>
                         <link>http://jgroups.org/javadoc</link>
+                        <!-- for javax.batch in the JSR 352 modules -->
+                        <link>https://docs.oracle.com/javaee/7/api/</link>
+                        <link>http://docs.jboss.org/jberet/${jberetVersion}/javadoc/jberet-core/</link>
```

Comment:
It will probably be useless if/when you only include the API/SPI in the javadoc but if one day we want to go back to the current setup, at least it will be comprehensive.